src/components/modal/ModalDialog.js

```diff
@@ -116,8 +116,8 @@ class ModalDialog extends React.Component<Props> {
                     <h2 className="modal-title" id={`${this.modalID}-label`}>
                         {title}
                     </h2>
+                    <>{this.renderCloseButton()}</>
                 </div>
-                {this.renderCloseButton()}
```

Comment:
What's the reason for moving this?

Reply:
So we can unify the header title and the close button in the same div. Currently the Close button was located outside the header along with the rest of the modal content.

Reply:
Gotcha, wasn't sure if this was correct semantically but I think it should be fine.

However, it looks like there are some `modal-header` usages in EUA that have `display: none`. Moving the close button inside the header will now hide the button for these modals (e.g. AnnotationsEngagementModal.scss, PaywallUpsellModal.scss).

I think we'll have to keep this on the outside so those modals still have a close button but no header. Otherwise, we'll have to update the modal stylesheets in the parent apps (EUA, AC, Dev Console).